docker-compose.yaml

```diff
@@ -20,11 +20,21 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    ports:
-      - 8020:8020
     volumes:
       - ./project/:/project/
       - ./main/:/main/
       - ./img/:/img/
     env_file:
       - .env.dev
+  nginx:
+    container_name: django-starter-nginx
+    image: nginx:1.19.0-alpine
+    ports:
+      - "8020:80"
+    volumes:
+      - ./static:/code/static
+      - ./media:/code/media
+      - ./nginx:/etc/nginx/conf.d/
+    depends_on:
+      - hub
```

Comment:
Can we change the name `hub` in this PR as well? I don't think we need it to be called hub, that was just a leftover from the project I adapted it from. It makes more sense to me to call it something like `django-project` for this generic starter, and encourage users to change it when they use this project.

Reply:
Done